sequencing/vcf_qc/dxapp.json

```diff
@@ -3,35 +3,108 @@
   "title": "VCF QC",
   "summary": "Apply Recalibration and hard filters to VCF file",
   "dxapi": "1.0.0",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
-      "name": "vcf_fn",
+      "name": "fn_vcfgz",
       "label": "VCF File",
       "class": "array:file",
       "optional": false,
-      "patterns": [
-        "*.vcf.gz"
-      ]
+      "patterns": ["*.vcf.gz"],
+      "help": "Array of VCF.gz files."
     },
     {
-      "name": "vcfidx_fn",
+      "name": "fn_vcfidx",
       "label": "VCF Index File",
       "class": "array:file",
       "optional": false,
-      "patterns": [
-        "*.tbi"
+      "patterns": ["*.tbi", "*.vcf.gz.tbi"],
+      "help": "Array of VCF.gz.tbi index files."
+    },
+    {
+      "name": "gatk_jar",
+      "label": "GATK jar file",
+      "class": "file",
+      "optional": false,
+      "patterns": ["GenomeAnalysisTK*.jar"],
+      "help": "The GenomeAnalysisTK.jar file."
+    },
+    {
+      "name": "genome_fasta",
+      "label": "Reference genome",
+      "help": "A FASTA file of reference genome.",
+      "class": "file",
+      "patterns": ["*.fasta","*.fa","*.fasta.gz","*.fa.gz"],
+      "optional": false,
+      "suggestions": [
+        {
+          "project": "project-FGqF5B00ByvPXG4G87yQy23G",
+          "path": "/GATK/resources",
+          "region": "aws:us-east-1",
+          "name": "dx_ritchie_collaboration: AWS US-east"
+        },
+        {
+          "project": "project-FGvp3009BJq40pkZ683v3754",
+          "path": "/GATK/resources",
+          "region": "aws:westus",
+          "name": "dx_ritchie_collaboration: Azure West US"
```

Comment:
Same as above. Not sure about the name used here. We (Ritchie lab and DNAnexus) may need to update to a generic name before paper submission and app publishing.